src/openMVG/sfm/sfm_data_io_gt.hpp

```diff
@@ -119,35 +126,47 @@ static bool read_Strecha_Camera(const std::string & camName, Pinhole_Intrinsic &
 
 /**
 @brief Reads a set of Pinhole Cameras and its poses from a ground truth dataset.
-@param[in] functorPointer, to the function which can handle the trajectory format. Example: &read_openMVG_Camera
 @param[in] sGTPath, the directory where the camera files are located.
```

Comment:
Instead of giving the path to the directory containing the `.camera` and then do a `folder_down` and a `folder_up` it is better to give as input the base path and then build the search paths adding "images" and "gt_dense_cameras". Explain this also in the doc.